os/arch/arm/src/armv7-r/arm_assert.c

```diff
@@ -204,7 +205,7 @@ static int is_text_address(unsigned long programCounter)
  * Below API works if there is existance of System.map file in rom fs
  ****************************************************************************/
 #ifdef CONFIG_FS_ROMFS
-void get_symbol(unsigned long search_addr, char *buffer)
+int get_symbol(unsigned long search_addr, char *buffer)
```

Comment:
This code is under CONFIG_FS_ROMF, but it is nothing to do with ROMFS. 
Shall we define a config for callstack symbol and place the corresponding code under this config?
CONFIG_FS_ROMFS can be used only to check whether /rom mount point is present or not. 
Rest of the logic is about searching symbol name in a file by using "address" as key.

Reply:
Junmin raise a pr for this. That is what manohara want to do. And me also.